invenio_deposit/links.py

```diff
@@ -55,6 +56,14 @@ def _url(name, **kwargs):
                        **kwargs)
 
     links['files'] = _url('files')
+
+    if current_app.config.get('DEPOSIT_UI_ENDPOINT') is not None:
```

Comment:
@drjova can you store `current_app.config.get('DEPOSIT_UI_ENDPOINT')` in temporary variable to avoid double access to the config object?